docs/int/quickstart/quickstart-group.md

```diff
@@ -0,0 +1,118 @@
+
+### Getting setup with charon
+
+1. Clone the [charon-distributed-validator-node](https://github.com/ObolNetwork/charon-distributed-validator-cluster) template repository from Github, and `cd` into the directory.
+
+    ```sh
+    # Clone the repo
+    git clone https://github.com/ObolNetwork/charon-distributed-validator-node.git
+    
+    # Change directory
+    cd charon-distributed-validator-node/
+    ```
+
+1. Prepare the environment variables
+
+    ```sh
+    # Copy the sample environment variables
+    cp .env.sample .env
+    ```
+
+    For simplicities sake, this repo is configured to work with a remote Beacon node such as one from [Infura](https://infura.io/). 
+    
+    Create an Eth2 project and copy the `https` URL:
+
+    ![Example Infura API Endpoint](/img/example-infura-details.png)
+
+    Replace the placeholder value of `CHARON_BEACON_NODE_ENDPOINT` in your newly created `.env` file with this URL.
+
+1. Now create a private key for charon to use for its ENR
+
+    ```sh
+    # Create an ENR private key
+    docker run --rm -v "$(pwd):/opt/charon" ghcr.io/obolnetwork/charon:latest create enr
+    ``` 
+    :::caution
+    The ability to replace a deleted or compromised private key is limited at this point. Please make a secure backup of this private key if this distributed validator is important to you.  
+    :::
+    This command will print your charon client's ENR to the command line. It should look something like:
+    ```
+    enr:-JG4QAgAOXjGFcTIkXBO30aUMzg2YSo1CYV0OH8Sf2s7zA2kFjVC9ZQ_jZZItdE8gA-tUXW-rWGDqEcoQkeJ98Pw7GaGAYFI7eoegmlkgnY0gmlwhCKNyGGJc2VjcDI1NmsxoQI6SQlzw3WGZ_VxFHLhawQFhCK8Aw7Z0zq8IABksuJEJIN0Y3CCPoODdWRwgj6E
+    ```
+    This record identifies your charon client no matter where it communicates from across the internet. It is required for the following step of creating a set of distributed validator private key shares amongst the cluster operators. 
+
+### Performing a Distributed Validator Key Generation Ceremony
+
+To create the private keys for a distributed validator securely, a Distributed Key Generation (DKG) process must take place.
+
+1. After gathering each operators ENR and setting them in the `.env` file, one operator should prepare the ceremony with `charon create dkg`
+    ```sh
+
+    # First set the ENRs of all the operators participating in DKG ceremony in .env file as CHARON_OPERATOR_ENRS
+
+    # Create .charon/cluster-definition.json to participate in DKG ceremony
+    docker run --rm -v "$(pwd):/opt/charon" --env-file .env ghcr.io/obolnetwork/charon:latest create dkg
+    ```
+1. The operator that ran this command should distribute the resulting `cluster-definition.json` file to each operator.
+
+1. At a pre-agreed time, all operators run the ceremony program with the `charon dkg` command
+    ```sh
+    # Copy the cluster-definition.json file to .charon
+    cp cluster-definition.json .charon/
```

Comment:
i think we don't need this, if everyone runs the commands as mentioned then `create dkg` command by default creates `cluster-definition.json` inside .charon

Reply:
If everyone ran `charon create dkg`, wouldn't they all have different UUIDs in the files and shouldn't the charon nodes fail the handshake? It has to be one cluster-definition.json shared to everyone, right?